python/cugraph/cugraph/community/egonet.py

```diff
@@ -95,6 +95,13 @@ def ego_graph(G, n, radius=1, center=True, undirected=None, distance=None):
 
     result_graph = type(G)(directed=G.is_directed())
 
+    if not G.edgelist.weights:
```

Comment:
C++ egonet does not require weights.  Not sure we should require that in python.

Reply:
I get a `segmentation fault` when running the CAPI tests